libpod/runtime_volume_linux.go

```diff
@@ -67,9 +67,20 @@ func (r *Runtime) newVolume(ctx context.Context, options ...VolumeCreateOption)
 	if volume.config.Driver == define.VolumeDriverLocal {
 		logrus.Debugf("Validating options for local driver")
 		// Validate options
-		for key := range volume.config.Options {
-			switch key {
-			case "device", "o", "type", "UID", "GID", "SIZE", "INODES":
+		for key, val := range volume.config.Options {
+			switch strings.ToUpper(key) {
+			case "DEVICE":
+				switch volume.config.Options["type"] {
+				case "bind":
+					if _, err := os.Stat(val); err != nil {
+						return nil, errors.Wrapf(err, "invalid volume option %s for driver 'local'", key)
+					}
+				case "tmpfs":
+					if val != "tmpfs" {
```

Comment:
Do we enforce this? I think this might break existing volumes, tmpfs used to be able to use any string here

Reply:
Removed the check.